src/neoExpressHelper.ts

```diff
@@ -0,0 +1,209 @@
+import * as childProcess from 'child_process';
+import * as shellEscape from 'shell-escape';
+
+class StringResult {
+    private constructor(public isError: boolean, public output: string) {
+    }
+
+    public static Error(output: string) {
+        return new StringResult(true, output);
+    }
+
+    public static Success(output: string) {
+        return new StringResult(false, output);
+    }
+}
+
+class JsonResult {
+    private constructor(public isError: boolean, public result: any) {
+    }
+
+    public static Error() {
+        return new JsonResult(true, {});
+    }
+
+    public static Success(outputJson: string) {
+        return new JsonResult(false, JSON.parse(outputJson));
+    }
+}
+
+export class NeoExpressHelper {
+
+    public static async claimGas(
```

Comment:
Thinking we should add --json option (even if it does nothing at first) to express claim and transfer commands. THoughts? Opened https://github.com/neo-project/neo-express/issues/26 to track

Reply:
Yes, I can't start passing it in as an argument until after you add support for it though (or the command will error out with "Unrecognized option '--json'")

Reply:
Given #33, I think we should *not* do this

Reply:
agree